README.md

```diff
@@ -33,9 +49,15 @@ Below is a description of each action in the [`project.yaml`](./project.yaml). A
     -   Runs [`Stage1_data_cleaning.R`](./analysis/preprocess/Stage1_data_cleaning.R)
     -   Applies quality assurance rule and inclusion/exclusion criteria
     -   Outputted dataset is analysis ready
+-   `consort_{cohort}_midpoint6`    
+    -   Runs [`Stage1_data_cleaning.R`](./analysis/preprocess/Stage1_data_cleaning.R)
+    -   Used to elaborate consort table, using study population size after each quality assurance, inclusion and exclusion criterion is applied.
 -   `table1_{cohort}`
     -   Runs [`table1.R`](./analysis/table1.R) which calculates descriptive statistics for pre- and post-exposure events for all outcomes and subgroups.
     -   Used for Table 1 in the manuscript
+-   `extendedtable1_{cohort}`    
+    -   Runs [`extendedtable1.R`](./analysis/extendedtable1.R) which calculates descriptive statistics for pre- and post-exposure events for all outcomes and subgroups.
+    -   Used for Extended Table 1 in the manuscript
 -   `table2_{cohort}`
     -   Runs [`table2.R`](./analysis/descriptives/table2.R) which calculates pre- and post-exposure event counts and person days of follow-up for all outcomes and subgroups.
     -   Used for Table 2 in the manuscript
```

Comment:
If you're going to add action descriptions, could we add make model input, reusable action location, and make model output

Reply:
The description for such actions is already there (see lines 64-74).